src/buza/templates/404.html

```diff
@@ -0,0 +1,11 @@
+{% extends 'base.html' %}
+{% load static %}
+{% block title %}Page not found  404 {{ block.super }}{% endblock %}
+{% block head_extra %}
+    {{ block.super }}
+    <link rel="stylesheet" href="{% static 'buza/css/errors.css' %}">
```

Comment:
@sewagodimo: This `errors.css` file seems to be missing: is this one you added locally, but haven't committed yet?

Reply:
No, I didn't realise that was still there
I want to style the 404 error page and the 400 error pages but it'snot a priority for now